appveyor.yml

```diff
@@ -0,0 +1,33 @@
+version: 1.0.{build}
+image: Visual Studio 2019
+
+install:
+  - ps: |
+      $url = "https://github.com/AdoptOpenJDK/openjdk15-binaries/releases/download/jdk-15.0.1%2B9/OpenJDK15U-jdk_x64_windows_hotspot_15.0.1_9.zip"
+      $output = "openjdk15.zip"
+      Invoke-WebRequest -Uri $url -OutFile $output
+      7z x $output -oC:\openjdk15
+      Remove-Item $output
+      [Environment]::SetEnvironmentVariable("JAVA_HOME", "C:\openjdk15\jdk-15.0.1+9", "Machine")
+      refreshenv
+      $env:PATH = "$env:JAVA_HOME\bin;$env:PATH"
+      echo $env:JAVA_HOME
+      echo $env:PATH
+      java -version
+      javac -version
+
+build_script:
+  - cmd: gradlew.bat build --info
+
+test_script:
+  - cmd: gradlew.bat test --info
+
+cache:
+  - '%USERPROFILE%\.gradle\caches'
+  - '%USERPROFILE%\.gradle\wrapper'
+
+notifications:
+  - provider: Email
```

Comment:
I think we don't need Email or need to specify because now it is giving error as email is **`null`**

Reply:
You're right. Actually, notifications part can be emitted wholely